src/main/java/org/gridsuite/modification/server/dto/UpdateModificationGroupResult.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.modification.server.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Builder
+@Data
+@Schema(description = "Copy or move network modifications result")
+//TODO : remove this DTO when modificationFailures will not be needed anymore
+public class UpdateModificationGroupResult {
+
+    @Builder.Default
+    Optional<NetworkModificationResult> networkModificationResult = Optional.empty();
```

Comment:
Why using an Optional here ?

Reply:
Because now we return an Optional as result, check https://github.com/gridsuite/network-modification-server/pull/213